src/com/untamedears/PrisonPearl/PrisonPearlCommands.java

```diff
@@ -177,10 +180,34 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
         	return banAlt(sender, args);
         } else if (label.equalsIgnoreCase("ppunban")){
         	return unBanAlt(sender, args);
+        } else if (label.equalsIgnoreCase("ppwb")){
+        	if(sender.hasPermission("prisonpearl.wb")) {// sees if the players has the permission.
```

Comment:
And so you did, apogies. My comment applies to the whole file then. Might just go ahead and fix it, the style as it is within just this one file makes readability very poor, and is inconsistent with the project as a whole. My apologies for laying the fault at your feet, clearly it was whomever came before!